decidim-process-extended/app/views/decidim/participatory_processes/admin/participatory_processes/_form.html.erb

```diff
@@ -59,29 +62,22 @@
     </div>
 
     <div class="row">
-      <% if false %>
-        <div class="columns xlarge-6">
-          <%= form.translated :text_field, :developer_group %>
-        </div>
-      <% end %>
+      <div class="columns xlarge-6">
```

Comment:
`if false` ! it was never rendered! shouldn't we remove this tags?

same below